src/test/java/com/iexec/worker/tee/scone/LasServicesManagerTests.java

```diff
@@ -188,6 +192,20 @@ private void startLasService(String chainTaskId,
     }
     // endregion
 
+    // region createLasContainerName
+    @Test
+    void shouldCreateLasContainerNameWithProperCharLength() {
+        LasServicesManager lasServicesManager = new LasServicesManager(
+                sconeConfiguration, smsClientProvider, workerConfigService,
+                sgxService, dockerService);
+        ECKeyPair keyPair = ECKeyPair.create(new BigInteger(32, new Random()));
+        when(workerConfigService.getWorkerWalletAddress())
+                .thenReturn(Credentials.create(keyPair).getAddress());
+        Assertions.assertTrue(
+                lasServicesManager.createLasContainerName().length() < 64);
```

Comment:
With our naming convention, shouldn't we check for a fixed size length ?

Reply:
I can upgrade the test yes, but essential part is having anything below 64, hence the `< 64` assertion.
Do you see my point? What do you prefer?

Reply:
It was just to note that `createLasContainerName` is not fully covered.
If you want to merge as is, let's go with this

Reply:
Sorry, I'm not sure to understand, can you detail a bit?

Reply:
I understand your focus on enforcing the container name stays below the 64 characters threshold.

To really cover the space of acceptable values/names, the container name should always follow the "iexec-las-" + <wallet-address> pattern and the name will have a length around sixty something characters.
In this scope, the test you added partially covers what should be tested.

This is just a note and my idea was just to point that sometime in the future, we will need to be more precise on valid values coverage.
Right here, right now, we can merge with the current implementation if you prefer.

Reply:
Is it better on your side now https://github.com/iExecBlockchainComputing/iexec-worker/pull/470/commits/c94eb4cb504c162256c7ca72e561462ab1c5a6b8?